### ISSUE TITLE
When content is changed in a Froala editor (based on a textarea), coupled scope variable is never updated

### ISSUE THREAD

**Issue by stivni** [froala/angular-froala#80]
```
I experienced this behavior in both master and 2.1.0 tag.
This behavior can be checked with the shipped demo.
```

When initializing a Froala editor on a textarea with the Froala directive, the associated scope variable, provided in the ng-model directive, never gets updated.
In the demo this results in the "Rendered content" being always empty.

If you inspect the demo and make the associated textarea visible, you can see the contents of the textarea are being updated, but I guess the changes are never applied to scope: If I manually `scope.$apply()` everything works as expected.

If necessary, I can investigate further.

**Comment by cgallarno** [comment]
@stivni Can you confirm that this commit fixes your issue? 83e7e4e6dbd90139f984fc74d18b366dd16f93ea

**Comment by stefanneculai** [comment]
@stivni Could you please try the latest master or the PR 83e7e4e? Thank you.
